scripts/generate_swagger.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Why is this script needed when the doc is generated dynamically during docker build? Also why is swagger.json and swagger.yaml in git? When this isn't a spec-first framework they shouldn't be there. It creates extra maintenance steps with every API change to keep them in sync with actual implementation.